app/views/admin/rebate_forms/edit.html.haml

```diff
@@ -1,4 +1,4 @@
-%h1 Editing admin_rebate_form
+%h1="Application ##{@rebate_form.id}"
```

Comment:
This doesn't mean much to the user. How about using the valuation id and the year?